### PR TITLE
Added optional isPrivate to trader details

### PR DESCRIPTION
**Pull request by paulhargrave**
### Type of change
- [ ] Breaking change
- [X ] Non-breaking change
- [ ] Cosmetic change

### Description
Jira link: [Jira title](https://jira.tools.tax.service.gov.uk/browse/ARSSTB-151)

Added in an Optional isPrivate to trader details in the Application and ApplicationRequest

### Attachments or Screenshots
_Add any screenshots (if relevant) here_

### CODE REVIEW

File: app/uk/gov/hmrc/advancevaluationrulings/models/application/Attachment.scala [hunk 1]
```diff
@@ -16,10 +16,11 @@
 
 package uk.gov.hmrc.advancevaluationrulings.models.application
 
+import com.fasterxml.jackson.annotation.JsonIgnore
 import play.api.libs.json.{Json, OFormat}
 
 final case class Attachment(
-  id: Long,
+  @JsonIgnore id: Long,
```
Comment (paulhargrave) [inline app/uk/gov/hmrc/advancevaluationrulings/models/application/Attachment.scala:23]: Actually I didn't mean to commit this. Will do another commit after standup

**Review by aaocs — APPROVED** [review]
